recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
 {% set name = "holoviews" %}
-{% set version = "1.14.8" %}
+{% set version = "1.15.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holoviews-{{ version }}.tar.gz
-  sha256: 6c365599a2cb16793bb627c9b5c5430982bb591c9de8885002ff91b539b61133
+  sha256: 312e60d24406c3a7aec46261836b38dba2585e366bccb45e47479504805eb5ba
 
 build:
   number: 0
-  noarch: python
+  # We cannot build a recent version of panel due to missing nodejs on s390x
+  skip: True  # [s390x]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
Could you please add skip `py<37`? https://github.com/holoviz/holoviews/blob/2d2b72809eb2e2d75f1e26f3fad97892b2d6a720/setup.py#L151